source/_components/leafspy.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: "Leaf Spy"
+description: "Instructions on how to setup Leaf Spy to post data to Home Assistant."
+date: 2019-05-04 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: leafspy.png
+ha_category: Car
+ha_iot_class: Local Push
+ha_release: 0.95
+---
+
+Leaf Spy is an application for Android and iOS giving Nissan Leaf owners a better insight into their car and supports pushing certain data directly to Home Assistant. It can be set up via the integrations panel in the configuration screen.
```

Comment:
```suggestion
Leaf Spy is an application for Android and iOS, giving Nissan Leaf owners better insight into their car and supports pushing certain data directly to Home Assistant. It can be set up via the integrations panel in the configuration screen.
```